ci/aws-analytical-env/shared/meta.yml

```diff
@@ -349,3 +350,184 @@ meta:
 
               jq < output.json
               jq -eC "if .errorMessage? then error(.errorMessage) else true end" < output.json
+
+    terraform-bootstrap-crown:
+      task: terraform-bootstrap-crown
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli.repository))
+            version: ((docker-awscli.version))
+            tag: ((docker-awscli.version))
```

Comment:
should these not be `docker.docker_awscli_repository`, `docker.docker_awscli_version`?

Reply:
Previously was using `repository: dwpdigital/jinja-yaml-aws` but this failed during run. Had a chat with @dan-hill2802  he suggested to use `docker-awscli.repository`